gpkit/nomials.py

```diff
@@ -454,9 +489,14 @@ def __pow__(self, other):
 
 
 class Constraint(Posynomial):
-    '''
-    TODO: Add docstring
-    '''
+    """A constraint of the general form monomial > posynomial
+    Stored internally (exps, cs) as a single Posynomial (1 >= self)
+    Usually initialized via operator overloading, e.g. cc = y**2 >= 1 + x
+    Additionally stores input format (lhs vs rhs) in self.right and self.left
+    Form is self.left >= self.right.
```

Comment:
Implemented in d00ef9654edc515d04203e416dd01f6583f5add8. I went with the standard form `left <= right` for all operator-overloaded initializations, in keeping with the notation/syntax of the Boyd GP tutorial.